docs/consolecontrol.md

```diff
@@ -0,0 +1,112 @@
+---
+title: ConsoleControl function
+description: Performs special control operations for console.
+author: zadjii-msft
+ms.author: migrie
+ms.topic: article
+keywords: console, character mode applications, command line applications, terminal applications, console api
+f1_keywords:
+- consoleapi/ConsoleControl
+- wincon/ConsoleControl
+- ConsoleControl
+MS-HAID:
+- '\_win32\_consolecontrol'
+- 'base.consolecontrol'
+- 'consoles.consolecontrol'
+MSHAttr:
+- 'PreferredSiteName:MSDN'
+- 'PreferredLib:/library/windows/desktop'
+ms.assetid: 5a9f7b18-cdea-4041-a377-5532d30e0105
+
+topic_type:
+- apiref
+api_name:
+- ConsoleControl
+api_location:
+- User32.dll
+api_type:
+- DllExport
+---
+
+# ConsoleControl function
+
+Performs special kernel operations for console host applications. This includes: reparenting the console window, allowing the console to give launched applications foreground rights, and terminating attached processes. 
```

Comment:
```suggestion
Performs special kernel operations for console host applications. This includes reparenting the console window, allowing the console to pass foreground rights on to launched console subsystem applications and terminating attached processes. 
```

reword 2, kill oxford comma 3

Reply:
(I reworded 2 because it sounds too much like "the console can break all of user32's foreground rules")

Reply:
Maybe "kernel operations" -> "Windows subsystem operations" (not all of win32 is in the kernel)